src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -198,15 +195,34 @@ private void AddPackageItemsToDictionary(ref Dictionary<string, PackageItem> pac
             }
         }
 
-        // A versions file is of the form https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt
+        private static async System.Threading.Tasks.Task<string> GetFileAsync(string uri)
+        {
+            string contents = string.Empty;
+            using (var stream = (File.Exists(uri)) ? File.OpenRead(uri) : await new HttpClient().GetStreamAsync(uri))
+            using (StreamReader reader = new StreamReader(stream))
+            {
+                contents = await reader.ReadToEndAsync();
+            }
+            return contents;
+        }
+
+        // A versions file is of the form https://raw.githubusercontent.com/dotnet/versions/master/build-info/dotnet/corefx/master/Latest_Packages.txt
```

Comment:
I see... and one thing I think I didn't have right was whether you were going to link [`Latest_Packages.txt`](https://github.com/dotnet/versions/blob/64d27bdd68ba6f9f1d62c7eaf11395090eed8f04/build-info/dotnet/corefx/release/1.0.0/Latest_Packages.txt) or [`LKG_Packages.txt`](https://github.com/dotnet/versions/blob/64d27bdd68ba6f9f1d62c7eaf11395090eed8f04/build-info/dotnet/corefx/release/1.0.0/LKG_Packages.txt). I was thinking Latest due to the comment and PR description, but LKG makes sense and makes me not concerned about repeatability. (Now I remember LKG is what you added in https://github.com/dotnet/corefx/pull/12382.)